pkg/domain/route.go

```diff
@@ -0,0 +1,6 @@
+package domain
+
+type Route struct {
```

Comment:
please rename to something that does not conflict with route as that is used for handlers/endpoints. maybe TransformerValue? and add the float64 to this struct as you will need to convert from a iot-message-bus.Route -> this struct to use the sdk. This struct would also be used in TDS to revert to the raw value as well. "Route" is specific to iot-message-bus